panther_analysis_tool/main.py

```diff
@@ -92,6 +92,15 @@
     SCHEDULED_RULE: RULE_SCHEMA,
 }
 
+SET_FIELDS = [
```

Comment:
This was populated w/ the struct used for DDB `tableItem`, specifically all of the fields that have `dynamodbav:"stringset"`

Reply:
`ResourceTypes` in the spec files are referenced by `LogTypes`

Reply:
I see, sounds good.

Reply:
In case you're curious, see the highlighted line:

![image](https://user-images.githubusercontent.com/71197790/115794900-dc3ad180-a39c-11eb-9b23-b98a8b5aa8f6.png)